_data/navbars/security.yml

```diff
@@ -76,8 +76,8 @@ section:
     path: /security/high-connection-workloads
   - title: Defend against DoS attacks
     path: /security/defend-dos-attack
-- title: Try out the pod-to-pod encryption tech preview
-  path: /security/try-node-to-node-encryption
+- title: Pod-to-pod encryption
+  path: /security/pod-to-pod-encryption
```

Comment:
Also need to add an entry to `netlify/_redirects` file to redirect from the old URL to the new one.